library/core/src/panic/panic_info.rs

```diff
@@ -62,6 +65,12 @@ impl<'a> PanicInfo<'a> {
         self.payload = info;
     }
 
+    #[unstable(feature = "error_in_panic", issue = "none")]
+    /// The lower-level source of this panic, if any.
```

Comment:
It might be good to explicitly call out that this is about an `Error`. Maybe something like:
```suggestion
    /// The [`Error`] that caused this panic, if known.
```